polonius-engine/src/output/datafrog_opt.rs

```diff
@@ -47,7 +47,9 @@ pub(super) fn compute<Region: Atom, Loan: Atom, Point: Atom>(
 
         // static inputs
         let cfg_edge = iteration.variable::<(Point, Point)>("cfg_edge");
-        let killed = all_facts.killed.into();
+        let cfg_edge_rel = Relation::from(all_facts.cfg_edge.iter().map(|&(p, q)| (p, q)));
```

Comment:
For posterity's sake (as you know this): I don't think we need the CFG edges as a `Variable` anymore with the leapjoin